src/pages/juniorPromise/components/TimeSelectionButton.tsx

```diff
@@ -0,0 +1,65 @@
+import styled from '@emotion/styled';
+import { ButtonCheckIc } from '../../../assets/svgs';
+import { TIME_SELECTION_TITLE } from '../constants/constants';
+
+function TimeSelectionButton() {
+  const buttonValue = '7월 8일 13:00 ~ 13:20';
+  // const buttonValue = null;
+
+  return (
+    <Container>
+      {TIME_SELECTION_TITLE.map((item, index) => (
+        <Wrapper key={index} isActive={buttonValue !== null}>
+          <Title2>{buttonValue ? buttonValue : item.title}</Title2>
```

Comment:
p5) 지나가던 행인입니다...!

```suggestion
          <Title2>{buttonValue || item.title}</Title2>
```
단락평가를 잘 이용하면 요로케 삼항연산자를 축약할 수도 있습니다!

Reply:
우왁 이런 피드백 너무 좋습니다... 새로운걸 알게 되었네요!!
반영했습니다 :>